content/blog/is_serverless_the_future_part_1/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Serverless: Is it Really the Future (Part 1)"
+date: 2021-03-30
+meta_desc: "Serverless is a popular because it's fast and cheap. This article discusses the pros and cons of serverless architecture."
```

Comment:
```suggestion
meta_desc: "Serverless is ~~a~~ popular because it's fast and cheap, but is it really? This article discusses the pros and cons of serverless architecture."
```